js/subsites-menu.js

```diff
@@ -9,6 +9,26 @@
       // was actually resized.
       let windowWidth = window.innerWidth;
 
+      let mobileBreakpointJS = 768;
+      // If there is a drupalSettings variable for the breakpoint, use that.
+      if (
+        drupalSettings.localgov_base &&
+        drupalSettings.localgov_base.mobileBreakpointJS
+      ) {
+        let mobileBreakpoint = drupalSettings.localgov_base.mobileBreakpointJS;
+
+        // 1. If it has 'px' at the end, remove it.
+        mobileBreakpoint = mobileBreakpoint.replace('px', '');
+
+        // 2. Convert it to a number.
+        mobileBreakpoint = parseInt(mobileBreakpoint, 10);
+
+        // 3. Check that the value is greater than 0.
+        if (mobileBreakpoint > 0) {
+          mobileBreakpointJS = mobileBreakpoint;
+        }
+      }
```

Comment:
I think that most/all of this could be handled in the existing `.theme` file functions (see below).

Reply:
Done.